### PR TITLE
Ovsdb test

### PR DESCRIPTION
**Pull request by hunchback**

### CODE REVIEW

File: pkg/ovsdb/dbserver.go [hunk 1]
```diff
@@ -19,14 +19,25 @@ import (
 	"github.com/ibm/ovsdb-etcd/pkg/types/_Server"
 )
 
+type DBServerInterface interface {
```
Comment (roytman) [inline pkg/ovsdb/dbserver.go:22]: I would suggest separating OVSDB methods, such as `Lock`, `Unlock`, GetSchema`, GetUUID` and others

File: pkg/ovsdb/dbserver.go [hunk 2]
```diff
@@ -259,3 +270,51 @@ func (con *DBServer) GetMarshaled(prefix string, columns []interface{}) (*[]map[
...
+}
+
+func (con *DBServer) GetUUID() string {
+	return con.uuid
+}
+
+type DBServerMock struct {
+}
+
+func NewDBServerMock() (DBServerInterface, error) {
```
Comment (roytman) [inline pkg/ovsdb/dbserver.go:285]: I'd prefer to put the Mock into a separate file

File: pkg/ovsdb/dbserver.go [hunk 3]
```diff
@@ -19,14 +19,25 @@ import (
 	"github.com/ibm/ovsdb-etcd/pkg/types/_Server"
 )
 
+type DBServerInterface interface {
+	Lock(ctx context.Context, id string) (bool, error)
+	Unlock(ctx context.Context, id string) error
+	AddSchema(schemaName, schemaFile string) error
```
Comment (roytman) [inline pkg/ovsdb/dbserver.go:25]: AddSchema and especially LoadServerData are part of the development scaffolding, they will be removed.

File: pkg/ovsdb/dbserver.go [hunk 4]
```diff
@@ -19,14 +19,25 @@ import (
...
+	LoadServerData() error
+	GetData(prefix string, keysOnly bool) (*clientv3.GetResponse, error)
+	GetMarshaled(prefix string, columns []interface{}) (*[]map[string]string, error)
```
Comment (roytman) [inline pkg/ovsdb/dbserver.go:28]: GetMarshaled, will probably  be removed too

**Comment by roytman** [comment]
we will have to add notifications, the process that starts as a result of etcd events

File: pkg/ovsdb/dbserver.go (hunk 1 above)
Comment (hunchback) [reply pkg/ovsdb/dbserver.go:22]: I rather this be done in future, so now I just wanted to make the *existing* code testable

File: pkg/ovsdb/dbserver.go (hunk 2 above)
Comment (hunchback) [reply pkg/ovsdb/dbserver.go:285]: this very C++ or Java but is not the norm in go, as is creates many many very small files (each with overhead of header includes etc.), ... I can show many examples to support this claim in leading go projects

File: pkg/ovsdb/dbserver.go (hunk 3 above)
Comment (hunchback) [reply pkg/ovsdb/dbserver.go:25]: but this is not related to the focus of this PR, this is future work which should be done but should not be mixed with this effort

File: pkg/ovsdb/dbserver.go (hunk 4 above)
Comment (hunchback) [reply pkg/ovsdb/dbserver.go:28]: same as above

**Pull request merged**